### PR TITLE
Add a simple traceconfigz page.

### PR DESCRIPTION
**Pull request by bogdandrutu**
To test this change:
./gradlew installDist && ./examples/build/install/examples/bin/ZPagesTester

Go to:
localhost:8080/traceconfigz

Missing tests, will be added later because this is a P0 for 0.6 and is relatively simple code.

### CODE REVIEW

File: contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TraceConfigzZPageHandler.java [hunk 1]
```diff
@@ -88,80 +98,86 @@ public void emitHtml(Map<String, String> queryMap, OutputStream outputStream) {
...
   private void maybeApplyChanges(Map<String, String> queryMap) {
-    if (queryMap.containsKey(CHANGE) && queryMap.get(CHANGE).equals(PERMANENT_CHANGE)) {
+    String changeStr = queryMap.get(CHANGE);
+    if (PERMANENT_CHANGE.equals(changeStr)) {
       TraceParams.Builder traceParamsBuilder = traceConfig.getActiveTraceParams().toBuilder();
-      if (queryMap.containsKey(QUERY_COMPONENT_SAMPLING_PROBABILITY)) {
-        double samplingProbability =
-            Double.parseDouble(queryMap.get(QUERY_COMPONENT_SAMPLING_PROBABILITY));
+      String samplingProbabilityStr = queryMap.get(QUERY_COMPONENT_SAMPLING_PROBABILITY);
+      if (samplingProbabilityStr != null && !samplingProbabilityStr.isEmpty()) {
```
Comment (sebright) [inline contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TraceConfigzZPageHandler.java]: Suggestion: You could use `com.google.common.base.Strings.isNullOrEmpty` here and below.

File: contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TraceConfigzZPageHandler.java [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+          + "</form>";
+
+  private static final String RESTORE_DEFAULT_FORM_BODY =
+      "<form action=/traceconfigz method=get>%n"
+          // Permanently changes table.
+          + "<b>Restore default</b> %n"
+          + "<input type=\"hidden\" name=\"%s\" value=\"%s\"></td>%n"
+          + "</br>%n"
+          // Submit button.
+          + "<input type=submit value=Start>%n"
```
Comment (songy23) [inline contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TraceConfigzZPageHandler.java]: Nit: this button could be named "Reset"

File: contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TraceConfigzZPageHandler.java [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+          + "<tr><td>MaxNumberOfLinks to</td>"
+          + "<td><input type=text size=10 name=%s value=\"\"></td> <td>(%d)</td>%n"
+          + "</table>%n"
+          // Submit button.
+          + "<input type=submit value=Start>%n"
+          + "</form>";
+
+  private static final String RESTORE_DEFAULT_FORM_BODY =
+      "<form action=/traceconfigz method=get>%n"
+          // Permanently changes table.
```
Comment (songy23) [inline contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TraceConfigzZPageHandler.java]: Restore changes?

File: contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TraceConfigzZPageHandler.java (hunk 1 above)
Comment (bogdandrutu) [reply contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TraceConfigzZPageHandler.java]: Done.

File: contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TraceConfigzZPageHandler.java (hunk 3 above)
Comment (bogdandrutu) [reply contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TraceConfigzZPageHandler.java]: Done.

File: contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TraceConfigzZPageHandler.java (hunk 2 above)
Comment (bogdandrutu) [reply contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TraceConfigzZPageHandler.java]: Done.

**Pull request merged**